client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -28,6 +28,7 @@ const UserPage = createClass({
 		const usernameWithS = this.props.username + (this.props.username.endsWith('s') ? `’` : `’s`);
 
 		const brews = _.groupBy(this.props.brews, (brew)=>{
+			brew.title = brew.title.trim();
```

Comment:
My only comment is I think we should move this "cleanup" step into the homebrew.api.js page when we first retrieve the brew list, so all the information coming into this user page is already "clean".

Similar to how we fill in missing values with defaults when getting old brews. Let's keep all the data cleanup at the point of retrieval where possible.

Reply:
I've just pushed a commit where the brew data cleanup is done in `app.js` instead; I haven't been able to test it yet but I expect that it will pass without issue.

Reply:
I've been able to test this on my local install now and it all appears to work as expected,